### PR TITLE
Fix arguments for super calls

### PR DESCRIPTION
**Pull request by MartinHjelmare**
- This slipped in, in the worker refactor PR https://github.com/oeway/ImJoy-Engine/pull/30.

### CODE REVIEW

File: imjoy/imjoyUtils3.py [hunk 1]
```diff
@@ -26,14 +26,14 @@ def __init__(self, pfunc, loop):
...
             self.loop.call_soon(self.set_result, result)
 
     def reject(self, error):
         """Reject promise."""
         if self._catch_handler or self._finally_handler:
-            super().reject(self, error)
+            super().reject(error)
         else:
             if error:
                 self.loop.call_soon(self.set_exception, Exception())
```
Comment (oeway) [inline imjoy/imjoyUtils3.py]: The error string should be passed into Exception right?

Comment (oeway) [reply imjoy/imjoyUtils3.py]: This line should be switched with the next line I think.

**Comment by oeway** [comment]
I have just fixed it, will merge now.

File: imjoy/imjoyUtils3.py (hunk 1 above)
Comment (MartinHjelmare) [reply imjoy/imjoyUtils3.py]: Sounds reasonable. :+1:

**Pull request merged**